src/main/scala/mesosphere/marathon/api/v2/QueueResource.scala

```diff
@@ -56,4 +79,9 @@ class QueueResource @Inject() (
 
 object QueueResource {
   val EmbedLastUnusedOffers = "lastUnusedOffers"
+
+  private val appNotFoundTasksQueue: PathId => String =
```

Comment:
lets use runSpec terminology

Reply:
Fixed in the name of the variable. Kept the error message as is.